bower.json

```diff
@@ -17,10 +17,20 @@
     "web component"
   ],
   "dependencies": {
-    "polymer": "^1.7.0"
+    "polymer": "Polymer/polymer#1.9 - 2"
   },
   "devDependencies": {
-    "d2l-demo-template": "~0.0.5"
+    "d2l-demo-template": "^1.0.1"
```

Comment:
I think this should be `^0.0.12 - 1.0.1` and then in the `1.x` variant have `0.0.12`? Similar to:
https://github.com/BrightspaceUI/colors/blob/master/bower.json